readme.md

```diff
@@ -25,7 +25,7 @@ You need to include hotkeys-js and ux shot:
 <script src="https://unpkg.com/uxshot"></script>
 
 <script>
-  window.uxshot({
+  window.uxshot.init({
```

Comment:
Why?

Reply:
explained IRL

This is an ~issue~ feature with babel 7